SolrNet/Impl/SolrPostConnection.cs

```diff
@@ -24,7 +24,9 @@ public PostSolrConnection(ISolrConnection conn, string serverUrl)
 			    this.serverUrl = serverUrl;
 		    }
 
-		    public string Post(string relativeUrl, string s)
+	      public string ServerUrl => serverUrl;
+
+	      public string Post(string relativeUrl, string s)
```

Comment:
Could you just run this file through `Format Document` (Ctrl + K + D) to clean up the indentation.

Reply:
Sorry, I have ReSharper and wasn't sure how far to go with formatting because my rules on tabs are slightly different.  If there's a next time, I'll be sure to do that.

Reply:
Perfect. I added a .editorconfig file, wonder if that has any influence. 
Again thanks for the contribution, which is more important than the code formatting.

Reply:
Thanks @adegroff for your contribution, and @gjunge for reviewing and green lighting.

I learned about expression bodied members also a few weeks ago when another project that I have stopped building as CI had Visual Studio 2013 and someone added a new property for an alerting functionality.